xrootd/protocol/handshake/handshake.go

```diff
@@ -26,6 +33,7 @@ type Request struct {
 	Reserved5 int32
 }
 
+// NewRequest forms a Request with predefined parameters.
```

Comment:
`with predefined parameters` is a bit cryptic and not very much helpful :)
instead, we could perhaps say that a `Request` is created to comply with the XRootD protocol version XXX.